packages/ckeditor5-dev-env/lib/utils/changelog.js

```diff
@@ -0,0 +1,249 @@
+				if ( note.title === 'BREAKING CHANGE' ) {
+					return 0;
+				}
+			}
+
+			if ( !hasNewFeatures && item.type === 'Feature' ) {
+				hasNewFeatures = true;
+			}
+		}
+
+		return hasNewFeatures ? 1 : 2;
+	},
+
+	/**
+	 * Parses command line arguments and returns them as a user-friendly hash.
+	 *
+	 * @returns {Object} options
+	 * @returns {String} options.token GitHub token used to authenticate.
+	 * @returns {Boolean} options.init Whether to create first release using this package.
+	 * @returns {Boolean} options.debug Whether to show additional logs.
+	 */
+	parseArguments( args = process.argv.slice( 2 ) ) {
+		const options = require( 'minimist' )( args, {
+			string: [
+				'token'
+			],
+
+			boolean: [
+				'init',
+				'debug'
+			],
+
+			default: {
+				init: false,
+				debug: false
+			}
+		} );
+
+		delete options._;
+
+		return options;
+	},
+
+	/**
+	 * Returns all changes between `currentTag` and `previousTag`.
+	 *
+	 * If `previousTag` is null, returns the whole file (first release).
+	 *
+	 * @param {String} currentTag
+	 * @param {String|null} previousTag
+	 * @returns {Promise}
+	 */
+	getLatestChangesFromChangelog( currentTag, previousTag ) {
+		currentTag = currentTag.replace( /^v/, '' );
+
+		return utils.getCurrentChangelog()
+			.then( ( changelog ) => {
+				changelog = changelog.replace( utils.changelogHeader, '' );
+
+				if ( previousTag ) {
+					previousTag = previousTag.replace( /^v/, '' );
+
+					changelog = changelog.match( new RegExp( `(## \\[${ currentTag }\\][\\w\\W]+)## \\[?${ previousTag }\\]?` ) )[ 1 ];
+				}
+
+				changelog = changelog.replace( new RegExp( `^## \\[?${ currentTag }\\]?.*` ), '' ).trim();
+
+				return Promise.resolve( changelog );
+			} );
+	},
+
+	/**
+	 * @returns {Promise}
+	 */
+	getCurrentChangelog() {
+		const changelogFile = path.resolve( utils.changelogFile );
+
+		return new Promise( ( resolve, reject ) => {
+			fs.readFile( changelogFile, 'utf-8', ( err, content ) => {
+				if ( err ) {
+					return reject( err );
+				}
+
+				resolve( content );
+			} );
+		} );
+	},
+
+	/**
+	 * @param {String} content
+	 * @returns {Promise}
+	 */
+	saveChangelog( content ) {
+		const changelogFile = path.resolve( utils.changelogFile );
+
+		return new Promise( ( resolve, reject ) => {
+			fs.writeFile( changelogFile, content, ( err ) => {
+				if ( err ) {
+					return reject( err );
+				}
+
+				resolve();
+			} );
+		} );
+	},
+
+	/**
+	 * Create a Github release.
+	 *
+	 * @param {String} token Token used to authenticate with GitHub.
+	 * @param {Object} options
+	 * @param {String} options.repositoryOwner Owner of the repository.
+	 * @param {String} options.repositoryName Repository name.
+	 * @param {String} options.version Name of tag connected with the release.
+	 * @param {String} options.description Description of the release.
+	 * @param {String} options.debug Whether to display additional logs.
+	 * @returns {Promise}
+	 */
+	createGithubRelease( token, options ) {
```

Comment:
This isn't a util. This is a specific task. It doesn't mean that it should end in the tasks.js file, but that it shouldn't be in the utils :P.

The utils hole is the last place where you want a file to end. And it must meet certain requirements, like having no requirements of its own other than other utils.